models/sipnet/R/met2model.SIPNET.R

```diff
@@ -155,12 +161,17 @@ met2model.SIPNET <- function(in.path, in.prefix, outfolder, start_date, end_date
       Tair <-ncdf4::ncvar_get(nc, "air_temperature")  ## in Kelvin
       Tair_C <- PEcAn.utils::ud_convert(Tair, "K", "degC")
       Qair <-ncdf4::ncvar_get(nc, "specific_humidity")  #humidity (kg/kg)
-      ws <- try(ncdf4::ncvar_get(nc, "wind_speed"))
-      if (!is.numeric(ws)) {
+      
+      # if we have wind speed.
+      if ("wind_speed" %in% nc.var.names) {
+        ws <- ncdf4::ncvar_get(nc, "wind_speed")
+      } else {
```

Comment:
👍 to this pattern. I in my recent similar change in write.configs you had a wishlist request to make the expected set configurable -- would that be nice to have here too? I could see either way: Maybe met variables are standard enough it would be less useful here?

Reply:
Thanks! I also liked your suggestion in #3554 of a NULL default to use all names. I'll tag you on a suggestion for that on the lines above.